pinakes/common/models/fields.py

```diff
@@ -0,0 +1,67 @@
+import functools
+import json
+
+from cryptography.fernet import Fernet, MultiFernet
+from django.conf import settings
+from django.core import checks
+from django.core.exceptions import ImproperlyConfigured
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+
+class EncryptedJsonField(models.Field):
```

Comment:
should you start with `models.TextField`?

Reply:
Nope, `TextField` adds an additional coercion which has to be disabled in this class. Adding a base class only to disable some of it's functionality doesn't have much sense to me.